README.md

```diff
@@ -31,11 +31,23 @@ totally fine.
 ## Questions
 * How can you change the ordering of the list? From ascending 
 to descending. Name at least two different methods.
+1.)  Flip comparison operators
+2.) flip the list
+  
 * Can the inner Node class be static? Justify your answer.
+  No
+
 * Can you name the generic placeholder `Placeholder` instead of
 `T`? If yes should you, do it?
+  Yes.
+  No i should use T (style)
+
 * Which benefits does this data structure have?
+  Always sorted
```

Comment:
Brings the benefit of faster searching. Divide and conquer algorithms like binary search with a runtime complexity of 
O(log n) are possible.